owasp-zap-ui-scan-template.yml

```diff
@@ -0,0 +1,53 @@
+parameters:
+  url: ''
+
+jobs:
+- job: Run_Owasp_Zap_Scan
+  pool: UKHO Ubuntu 1804
+
+  workspace:
+    clean: all
+
+  steps:
+  - task: CmdLine@2
+    inputs:
+      script: 'chmod 777 -R $(Build.ArtifactStagingDirectory)'
+    displayName: "Set chmod permissions (ArtifactStagingDirectory)"
```

Comment:
Would this read better as "Set permissions on Build.ArtifactStagingDirectory" - consistent use of "Build.ArtifactStagingDirectory"in the directory names